theme.toml

```diff
@@ -60,6 +60,11 @@ error = "apricot"
 info = "delta"
 hint = "silver"
 
+diff.added = "#35BF86"
+diff.removed.above = "#F22C86"
+diff.removed.below = "#F22C86"
+diff.modified = "#6F44F0"
```

Comment:
```suggestion
"diff.added" = "#35BF86"
"diff.removed.above" = "#F22C86"
"diff.removed.below" = "#F22C86"
"diff.modified" = "#6F44F0"
```

Reply:
I knew that, had done it with the other files, but somehow forgot to fix the default theme. fixed.